build/common.sh

```diff
@@ -424,7 +424,7 @@ function kube::build::build_image() {
 
   kube::build::update_dockerfile
   kube::build::set_proxy
-  kube::build::docker_build "${KUBE_BUILD_IMAGE}" "${LOCAL_OUTPUT_BUILD_CONTEXT}" 'false'
+  kube::build::docker_build "${KUBE_BUILD_IMAGE}" "${LOCAL_OUTPUT_BUILD_CONTEXT}" 'true'
```

Comment:
I assume we're not trying to pull every time, perhaps something else should ensure it exists as opposed to just pulling every time.

Have we analyzed why this was previously set to true? git history?

Reply:
> I assume we're not trying to pull every time, perhaps something else should ensure it exists as opposed to just pulling every time.

I see, but in that case the doc is wrong... if someone can enlighten me what pre-req step I may be missing, I'd be delighted. OTH seeing `FROM k8s.gcr.io/kube-cross:v1.12.5-1` in the respective `Dockerfile` does make it seem to me this was intended to pull 

> Have we analyzed why this was previously set to true? git history?

git blame shows d403c2c16893c98aaeb1ec30bf352b95f4006541 last touched that line, but it just moved a few lines up. Do you have any tricks up your sleeve I should learn how to to easily see the previous revision git blame until you see where it was originally introduced?